dom.bs

```diff
@@ -816,7 +818,14 @@ must return the values they were initialized to.
 The
 <dfn method for=Event>preventDefault()</dfn>
 method must set the <a>canceled flag</a> if the
-{{Event/cancelable}} attribute value is true.
+{{Event/cancelable}} attribute value is true and
+the <a>in passive listener flag</a> is unset.
+
+<p class="note no-backref">
+ User agents are encouraged to generate a console warning or other debugging
+ aid to help authors identify places where calls to {{preventDefault()}}
+ have no effect.
```

Comment:
@domenic started using "User agents are encouraged to log the precise cause in a developer console, to aid debugging." in HTML. I suggest we start from there.